app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -548,8 +548,8 @@ def clone_vm(raw_args)
         "numCPUs" => args[:cpus],
         "numCoresPerSocket" => args[:corespersocket],
         "memoryMB" => args[:memory_mb],
-        "datastore" => args[:volumes].first[:datastore],
-        "storage_pod" => args[:volumes].first[:storage_pod],
+        "datastore" => args[:volumes].empty? ? nil : args[:volumes].first[:datastore],
+        "storage_pod" => args[:volumes].empty? ? nil : args[:volumes].first[:storage_pod],
```

Comment:
This doesn't look like it's specific to hammer, would a clearer commit message be something like "fix vmware vm cloning when volumes are not present"?

Reply:
Yes, I agree I will change the commit message.